.ko.yaml

```diff
@@ -342,3 +346,8 @@ builds:
   ldflags:
   - -s -w
   - -X k8s.io/test-infra/gcsweb/pkg/version.VERSION={{.Env.VERSION}}
+- id: analysis
```

Comment:
nit: this is not really necessary for this image, the only reason why images are defined here is because they need the `version.VERSION` info